pkgs/applications/networking/ids/suricata/default.nix

```diff
@@ -1,13 +1,18 @@
 { stdenv
 , lib
 , fetchurl
+, clang
+, llvm
 , pkgconfig
 , makeWrapper
 , file
 , geoip
```

Comment:
if `geoip` isn't needed anymore remove it from the imports (`libmaxminddb` is used instead?)

Reply:
It is.
Done and moved `libmaxminddb` to maintain alphabetic order.